### PR TITLE
fix: complete remaining part of pricing page

### LINKED ISSUE #4741: Pricing Page: Create a pricing page with info in the central column
As the service gets more reliable we will be able to cover our cost through pro-features that users would like to pay for. Therefore please implement a pricing page at /pricing

Relevant examples of pricing pages are https://www.eventbrite.co.uk/organizer/pricing/ and https://www.papercall.io/pricing.

* [x] Change the pricing link on the footer to the page.
Screenshot from 2020-09-03 18-43-45

* [ ] On the top menu add the menu item "Pricing" behind the "Browse Events" link **but** only show it for non-logged in users. For logged in users do not show the "Pricing" tab. It is sufficient to have it in the footer for them.
Screenshot from 2020-09-03 18-54-58

* [x] Add the following to the Page Content

  * [ ] Pricing (header)
Screenshot from 2020-09-03 18-41-58

  * [x] Find the plan that's right for you.
Screenshot from 2020-08-11 15-32-54

  * [x] Add table similar to Eventbrite. Use the details as provided in the spreadsheet here: https://docs.google.com/spreadsheets/d/1WTgOEtqmeGl-p2HCsBBNKJWC8Z_owwN8hpNoggezL4Y/edit?usp=sharing

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4741
scrnli_9_4_2020_10-01-39 PM
Screenshot from 2020-09-04 21-25-33

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/pricing.hbs [hunk 1]
```diff
@@ -1,4 +1,10 @@
+<div class="sixteen wide column lead {{if this.smallLead 'small'}}">
+  <Widgets::SafeImage class="background" @src="https://user-images.githubusercontent.com/56407566/92262388-a7a30400-eef8-11ea-9a63-8cf78ce4dfe0.png" />
```
Comment (iamareebjamal) [inline app/templates/pricing.hbs:2]: ??

Comment (maze-runnar) [reply app/templates/pricing.hbs:2]: @mariobehling has said to add a header image in issue description, so I added this one.

**Comment by mariobehling** [comment]
Please add an image header in a more standard way, e.g. compare to pages such as this one https://eventyay.com/e/16fa59c7/speakers

There is no space between the image and the header and the header image is not so big.

Screenshot from 2020-09-04 23-38-44

**Comment by mariobehling** [comment]
For the header image, please use this photo (height might have to be cropped): https://photos.google.com/share/AF1QipPEmvECPM6V7QHudeyqtCKvPLxUWbij9CnskJJ275drGgI6ckAnwpesQMwjtNGyBw/photo/AF1QipOvVzFFkaAVYx0d1OF2VbNC81jrYSTGWo4N66Q9?key=cGRoNk9MOXB6ZGhTQXlvSFBGc1JmOUw2dG9GXzdR

**Comment by mariobehling** [comment]
The lines of the table have different width somehow. Do you see that?

File: app/templates/pricing.hbs (hunk 1 above)
Comment (iamareebjamal) [reply app/templates/pricing.hbs:2]: This should be in assets, not uploaded to github

**Comment by iamareebjamal** [comment]
Line width issue is in Semantic UI

**Comment by maze-runnar** [comment]
scrnli_9_5_2020_7-02-22 PM

**Comment by iamareebjamal** [comment]
1.43 MB for an image is too much. Please use https://squoosh.app to reduce the image size and name the image `pricing-header`

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Please also ensure the image is full-width as the other images. Looks like there is some spacing on the left and right.

Screenshot from 2020-09-07 12-04-46

**Comment by maze-runnar** [comment]
> Please also ensure the image is full-width as the other images. Looks like there is some spacing on the left and right.
> 
> Screenshot from 2020-09-07 12-04-46

@iamareebjamal can u please help me with it, I have tried but it's not taking full width.

**Comment by iamareebjamal** [comment]
Apply `margin: 0` class from helpers file to `ui grid` element

**Comment by maze-runnar** [comment]
scrnli_9_7_2020_5-07-08 PM

**Comment by maze-runnar** [comment]
@iamareebjamal please review

**Comment by iamareebjamal** [comment]
Please add `margin-top` (mt) class to choose the plan header

**Comment by maze-runnar** [comment]
> Please add `margin-top` (mt) class to choose the plan header

Screenshot from 2020-09-07 18-05-49

**Comment by iamareebjamal** [comment]
Please use classes wherever possible. `mt-8`

File: app/templates/pricing.hbs [hunk 2]
```diff
@@ -1,4 +1,12 @@
-<div class="ui thirteen column center aligned container">
+<div class="public-event ui relaxed grid">
+  <div class="sixteen wide column lead {{if this.smallLead 'small'}}" style={{css max-height="300px"}}>
+    <Widgets::SafeImage class="background" @src="/images/pricing-header.jpg" />
+  </div>
+</div>
+<div class="ui thirteen column center aligned container style={{css margin-top="5px"}}">
```
Comment (iamareebjamal) [inline app/templates/pricing.hbs:6]:
```suggestion
<div class="ui thirteen column center aligned container mt-8">
```

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**